plugins/DevicesDetection/Commands/WarmDeviceDetectorCache.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+
+namespace Piwik\Plugins\DevicesDetection\Commands;
+
+use Piwik\DeviceDetector\DeviceDetectorCacheEntry;
+use Piwik\DeviceDetector\DeviceDetectorFactory;
+use Piwik\Plugin\ConsoleCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class WarmDeviceDetectorCache extends ConsoleCommand
+{
+    const OPTION_INPUT_FILE = 'input-file';
+    const OPTION_SKIP_HEADER = 'skip-header-row';
+    const OPTION_ROWS_TO_PROCESS = 'count';
+
+    private static $userAgentsPatternsToIgnore = array(
+        '/Amazon-Route53-Health-Check-Service[.]*/'
+    );
+
+    private $numCacheEntriesWritten = 0;
+
+    protected function configure()
+    {
+        $this->setName('devicedetector:warmcache');
```

Comment:
when I run the command, I get this error:

```
 [UnexpectedValueException]                                                                                                                                    
  RecursiveDirectoryIterator::__construct(/misc/useragents): failed to open dir: No such file or directory  
```

We could create the directory already in git and put a `.gitkeep` in there?